Dockerfile

```diff
@@ -0,0 +1,58 @@
+# syntax = docker/dockerfile:1
+
+# Adjust NODE_VERSION as desired
+ARG NODE_VERSION=18.17.1
+FROM node:${NODE_VERSION}-slim as base
+
+LABEL fly_launch_runtime="Next.js"
```

Comment:
@paradoxe35 is that required in Dockerfile? Because this same dockerfile can be used not only in Fly.io but in other providers, so we would prefer to keep the Dockerfile "clean" from anything fly-specific. If it's required, maybe we can specify it some other way in fly config or maybe we need to create 2 Dockerfiles, one for Fly and another "common"?

Reply:
Don't think it's required
Let me remove it and try to deploy 

@evereq

Reply:
Just removed that line `LABEL fly_launch_runtime="Next.js"`